tetrahedron/data/info.txt

```diff
@@ -3,4 +3,9 @@
 
```

Comment:
all the code should be placed immediately in the branch, not in the folder  tetrahedron

Reply:
if i understand correct our teacher)